.github/workflows/changelog.yml

```diff
@@ -22,4 +22,3 @@ jobs:
         uses: awharn/check_changelog_action@v1
```

Comment:
In order for [this action](https://github.com/awharn/check_changelog_action/blob/master/action.yml) to work correctly, I think it needs configuration to know this is a monorepo.

Instead of `yarnWorkspaces: true` we could set `lerna: true` since this repository has a lerna.json file 😋